src/Linters/UseStatementWIthoutKindLinter.php

```diff
@@ -0,0 +1,128 @@
+<?hh // strict
+/*
+ *  Copyright (c) 2017-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the MIT license found in the
+ *  LICENSE file in the root directory of this source tree.
+ *
+ */
+
+namespace Facebook\HHAST\Linters;
+
+use type Facebook\HHAST\{
+  EditableNode,
+  INamespaceUseDeclaration,
+  NamespaceUseClause,
+  NamespaceToken,
+  NameToken,
+  ScopeResolutionExpression,
+  SimpleTypeSpecifier,
+  TypeToken,
+  QualifiedName,
+};
+use namespace Facebook\HHAST;
+use namespace Facebook\TypeAssert;
+use namespace HH\Lib\{C, Keyset, Str};
+
+class UseStatementWithoutKindLinter
+  extends AutoFixingASTLinter<INamespaceUseDeclaration> {
+  <<__Override>>
+  protected static function getTargetType(
+  ): classname<INamespaceUseDeclaration> {
+    return INamespaceUseDeclaration::class;
+  }
+
+  <<__Override>>
+  public function getLintErrorForNode(
+    INamespaceUseDeclaration $node,
+    vec<EditableNode> $_context,
+  ): ?FixableASTLintError<INamespaceUseDeclaration> {
+    if ($node->hasKind()) {
+      return null;
+    }
+
+    return new FixableASTLintError(
+      $this,
+      "Use `use type` or `use namespace`",
```

Comment:
Isn't `use function` a thing too?

Reply:
Yep, however `use Foo` is equivalent to `use type Foo; use namespace Foo;` - it doesn't include `use function` or `use const`